locales/en/app.ftl

```diff
@@ -459,3 +459,305 @@ get-your-report-and-sign-up= Get your report and sign up for alerts.
+  to better protect your online accounts, and get alerted if your email address 
+  appears in a new breach.
+
+
+## How FXM Works
+how-fxm-works = How {-product-name} works
+
+how-fxm-1-headline = Conduct a basic search
+how-fxm-1-blurb = Search for your email address in public data breaches going 
+  back to 2007. This basic search will surface most data breaches, but not 
+  ones that contain sensitive personal information.
+
+how-fxm-2-headline = Sign up for breach monitoring
+how-fxm-2-blurb = Create a {-brand-fxa} to monitor your email for ongoing breaches. 
+  Once you’ve verified your email, you’ll also receive a full report of past breaches 
+  including sensitive breaches. 
+
+how-fxm-3-headline = Get notifications in your browser
+how-fxm-3-blurb = If you use {-brand-name}, you’ll receive a notification if you visit a 
+  site that’s been breached. Find out right away if you’ve been in the breach and 
+  what you can do about it.
+
+
+wtd-after-website = What to do after a website breach
+wtd-after-data-agg = What to do after a data aggregator breach
+
+what-is-data-agg = What is a data aggregator?
+what-is-data-agg-blurb = Data aggregators, or data brokers, collect information from public 
+  records and buy it from other companies. They compile this data to sell it to companies 
+  for marketing purposes. Victims of these breaches are less likely to experience financial 
+  fraud, but hackers could use this data to impersonate or profile them.
+
+protect-your-privacy = Protect your online privacy
+no-pw-to-change = Unlike a website breach, there’s no password to change.
+link-steps-to-protect = Steps to take to protect your identity online
+
+avoid-personal-info = Avoid using personal info in passwords
+avoid-personal-info-blurb = It’s easy to find birthdays, addresses, and family member names online. Keep it out of your passwords.
+
+## What to do after data breach tips
+
+change-pw = Change your password
+even-for-old = Even for old accounts, it’s important to update your password.
+
+make-new-pw-unique = Make the new password different and unique
+strength-of-your-pw = The strength of your passwords directly impacts your online security.
+create-strong-passwords = How to create strong passwords
+
+stop-reusing-pw = Stop reusing the same passwords
+create-unique-pw = Create unique passwords and save them somewhere safe, like a password manager.
+five-myths = 5 myths about password managers
+
+show-all-breaches = Show All Breaches
+
+create-a-fxa = Create a {-brand-fxa} for your full report of breaches and to get alerts.
+
+feat-security-tips = Security tips to protect your accounts
+feat-sensitive = Advanced search in sensitive breaches
+feat-enroll-multiple = Enroll multiple emails in breach monitoring
+
+sign-up-for-fxa = Sign up for a {-brand-fxa}
+
+
+appears-in-x-breaches =
+  { $breachCount ->
+        [one] Appears in { $breachCount } known breach.
+       *[other] appears in { $breachCount } known breaches.
```

Comment:
Not sure if this should be uppercase or lowercase. This was the only context I could find, and it looks like the string is prefixed by the user's email address. Should the email address be in this ftl string in case it should appear in a different order in different locales?

https://github.com/mozilla/blurts-server/blob/856d840348caad99c36e0f5abe260d53dca66e7c/views/partials/email-card.hbs#L3-L12

Reply:
Yes, the email should be part of the string. My point was that the two forms should be consistent.